libgap.pc.in

```diff
@@ -0,0 +1,11 @@
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+libdir=@libdir@
+includedir=@prefix@/include
+
+Name: @PACKAGE_NAME@
+Description: Library containing the kernel of GAP, a computer algebra system
```

Comment:
According to https://people.freedesktop.org/~dbn/pkg-config-guide.html and a bunch of *.pc files I looked at, we can (and then probably should) specify an URL, too. So maybe let's do this:
```suggestion
Description: Library containing the kernel of GAP, a computer algebra system
URL: https://www.gap-system.org
```